ckanext/data_qld/templates/datarequests/close.html

```diff
@@ -1,16 +1,20 @@
 {% ckan_extends %}
+{% set datasets = datasets or c.datasets %}
```

Comment:
can you add a comment that this c.datasets if pre 2.9 backwards compatability.

Reply:
It's more for compatibility with previous versions of ckanext-datarequests, but sure.